egs2/zeroth_korean/asr1/local/update_segmentation.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright 2017 Lucas Jo (Atlas Guide)
+# Apache 2.0
+
+# do this when the segmentation rule is changed
+dataDir=$1
+lmDir=$2
+
+exists(){
+	command -v "$1" >/dev/null 2>&1
+}
+
+# check morfessor installation 
+if ! exists morfessor; then
+	echo "You appear to not have Morfessor installed, either on your path."
```

Comment:
Could you make tools/installers/install_morfessor.sh and add the path to path.sh?
Now path.sh is symbolic link, so you also need to copy it.

```sh
# in path.sh

export PATH=<morfessor>:$PATH
if ! command -v morfessor >/dev/null 2>&1; then
	echo "You appear to not have Morfessor installed, either on your path."
        echo "Run tools/installers/install_morfessor.sh"
	return 1
fi
```

Reply:
I updated the recipe.